README.md

```diff
@@ -4,5 +4,13 @@ To run, follow these steps:
 
 1. Download ZIP or clone in GitHub
 2. Navigate to `liquid-gh-pages` folder or checkout `gh-pages` branch
-3. Run `bundle exec jekyll serve`
-4. Open `http://127.0.0.1:4000/liquid/` in your browser
+3. Run `bundle install` *(First time only. If it didn't work, see below.)*
+4. Run `bundle exec jekyll serve`
+5. Open [`http://127.0.0.1:4000/liquid/`](http://127.0.0.1:4000/liquid/) in your browser
+
+If you're on Windows and don't have `bundle`:
+
+1. If you don't have Ruby, install it *with Devkit* from [Ruby Downloads](https://rubyinstaller.org/downloads/).
+   * Be sure to run `ridk install` as the last step. Press *Enter* at the prompts.
+2. Run `gem install bundler`
```

Comment:
This isn't windows specific.  It is more so ruby specific, since bundler comes with ruby as of ruby 2.6.